src/containerapp-compose/setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = '0.1.0'
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = ['azure-cli-core', 'pycomposefile>=0.0.26']
```

Comment:
Please do not add `azure-cli-core` to the dependency, I vaguely remember that this led to package conflicts in the CI pipeline 
```suggestion
DEPENDENCIES = ['pycomposefile>=0.0.26']
```

Reply:
Removed